svo (install the zip, not me).xml

```diff
@@ -15251,9 +15251,11 @@ svo.valid.remove_unknownany(&quot;clumsiness&quot;)</script>
                         <colorTriggerBgColor>#000000</colorTriggerBgColor>
                         <regexCodeList>
                             <string>You lash out clumsily.</string>
+                            <string>^You lash out at (\w+) with (.+), but miss\.$</string>
```

Comment:
@EldFitheach good points all around. This seems to be not reliable enough as an indicator to rely on. @johndahlstrom @vadi2 any other thoughts? Otherwise I'd close this without merging.